src/build.rs

```diff
@@ -14,6 +17,7 @@ use crate::heading;
     after_help = "Run `cargo help build` for more detailed information."
 )]
 #[group(skip)]
+#[cfg_attr(feature = "serializable", derive(Deserialize, Serialize))]
 pub struct Build {
     #[command(flatten)]
     pub common: CommonOptions,
```

Comment:
I guess we also need to add `serde(flatten)`?

Reply:
I've updated the feature name everywhere.

Reply:
I've added `serde(flatten)` and `serde(default)` to allow for missing fields.